pkg/config/config.go

```diff
@@ -508,18 +510,19 @@ func (c *Config) BlockstoreAzureParams() (blockparams.Azure, error) {
 	if c.Blockstore.Azure.AuthMethod != "" {
 		logging.ContextUnavailable().Warn("blockstore.azure.auth_method is deprecated. Value is no longer used.")
 	}
-	if c.Blockstore.Azure.ChinaCloud {
-		logging.ContextUnavailable().Warn("blockstore.azure.china_cloud is enabled. lakeFS will only function on Azure China Cloud")
+	if c.Blockstore.Azure.ChinaCloudDeprecated {
+		logging.ContextUnavailable().Warn("blockstore.azure.china_cloud is deprecated. Value is no longer used. Please pass Domain = 'blob.core.chinacloudapi.cn'")
+		c.Blockstore.Azure.Domain = "blob.core.chinacloudapi.cn"
```

Comment:
I wouldn't warn here, I would panic. We're allowed to drop support for beta features. 
It was released only recently, it's easier to drop support now instead of later.

Reply:
I don't want to break backward compatibility to current users. We didn't advertise this as a Beta feature, I don't see this becoming a bigger problem in the future